elements/pl-drawing/mechanicsObjects.js

```diff
@@ -2037,12 +2037,7 @@ mechanicsObjects.byType['pl-arc-dimensions'] = class extends PLDrawingBaseElemen
 
           if (obj.label) {
               let dt = obj.endAngle-obj.startAngle;
-              let t;
-              if (dt >= 0) {
-                  t = obj.startAngle + dt/2;
-              } else {
-                  t = obj.startAngle + (360+dt)/2;
-              }
+              let t = obj.startAngle + dt/2;
```

Comment:
Are these changes relevant to the variable output?

Reply:
No. This is just for label offset in pl-arc-dimensions where in certain cases, the label offset is completely incorrect due to the else statement.

Reply:
@fmarrar it looks like you tangled this branch together with the one from #4788

Reply:
When you made the `add-sympy-tab` branch, it looks like you forgot to switch back to `master` first. That is, you were still on the `fix-arc-dimensions-label` branch when you made the `add-sympy-tab` branch, so `add-sympy-tab` includes all the changes from `fix-arc-dimensions-label`. To fix this, you should do:
```
git checkout master
git pull
git checkout add-sympy-tab
git rebase --onto master fix-arc-dimensions-label add-sympy-tab
git push --force origin add-sympy-tab
```

Reply:
Extremely sorry about that. I did -u when I pushed so I thought it would not conflict with the other branch.

Reply:
No worries. The problem isn't with the push, the problem was with the branches themselves on your local machine. But the above commands will fix it.

Reply:
Just did that. Again, incredibly sorry about that.

Reply:
No worries at all, we've all messed up with git before 😄